kubernetes/samples/scripts/domain-lifecycle/clusterStatus.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+
+function usage() {
+cat<<EOF
+
+Usage:
+  $(basename $0) [-n DomainNS] [-d DomainUID] [-c clsterName] [-m kubernetesCli] [-h]
+
+  This script dumps a table of WKO domain status
+  replica information for each given cluster.
+
+  Omit '-n' or set it to "" to indicate all namespaces.
+  Omit '-d' or set it to "" to indicate all domains in the given namespace(s).
+  Omit '-c' or set it to "" to indicate all clusters in the given domain(s) and the given namespace(s).
+  Use '-m' to specify a Kubernetes command line interface (defaults to 'kubectl').
+  Use '-h' to get this help.
+
+Examples:
+  - Dump cluster 'c1' in uid 'myuid' in ns 'myns': $(basename $0) -n myns -d myuid -c c1
+
+  - Dump all clusters in uid 'myuid' in ns 'myns': $(basename $0) -n myns -d myuid
+                                                   -or-
+                                                   $(basename $0) -n myns -d "" -c ""
```

Comment:
Th -or- is incorrect, it should have `-d myuid` instead of `-d ""`.  
Regardless I think we can just remove this particular -or- all-together - it's only there to demonstrate the use of `""` but the following sample command already demonstrates `""`.   Maybe even remove the following '-or-' too -- I put it in there myself, but, thinking about it some more, the demonstration of `""` is trivially helpful.